kimchi/src/circuits/constraints.rs

```diff
@@ -373,25 +374,71 @@ impl<F: PrimeField, G: KimchiCurve<ScalarField = F>, OpeningProof: OpenProof<G>>
 impl<F: PrimeField> ConstraintSystem<F> {
     /// evaluate witness polynomials over domains
     pub fn evaluate(&self, w: &[DP<F>; COLUMNS], z: &DP<F>) -> WitnessOverDomains<F> {
```

Comment:
Can you provide a bench for this method, please? And give us a before + after result?
It would be good for regressions and reproducibility.

Reply:
As discussed, it is not required for this PR, but if we want to provide continuous bench (as I think the team is working on?), that would be nice.

Reply:
I'll post the summary of all the prover criterion benchmarks tomorrow per commit, it should be OK for now? NB: even the current benchmarks still have A LOT of verifier noise (~1%, sometimes up to 3%, maybe it's just my machine), so the prover data is much more reliable.

Reply:
will post in the main thread